### PR TITLE
Tiny fix to an error message to mergeTrigger

### PR DESCRIPTION
**Pull request by kkaneda**
Log a new end key instead of a start key.

@vivekmenezes , I think this one is correct... Could you take a look?

### CODE REVIEW

File: storage/range_command.go [hunk 1]
```diff
@@ -1053,7 +1053,7 @@ func (r *Range) mergeTrigger(batch engine.Engine, merge *proto.MergeTrigger) err
 
 	if !r.Desc().EndKey.Less(merge.UpdatedDesc.EndKey) {
 		return util.Errorf("range end key is not less than the post merge end key: %s < %s",
-			r.Desc().EndKey, merge.UpdatedDesc.StartKey)
+			r.Desc().EndKey, merge.UpdatedDesc.EndKey)
```
Comment (vivekmenezes) [inline storage/range_command.go]: Is this meant to be
merge.UpdatedDesc.EndKey, r.Desc().EndKey)
to match the error message?

**Comment by vivekmenezes** [comment]
LGTM

File: storage/range_command.go (hunk 1 above)
Comment (kkaneda) [inline storage/range_command.go]: Sure, changed to `>=`.

**Comment by tamird** [comment]
:shipit:

**Pull request merged**